MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -1423,7 +1423,7 @@ public boolean recruitPerson(Person p, PrisonerStatus prisonerStatus, boolean gm
      * @param ignoreDice If true, skips the random roll and assigns a Bloodname automatically
      */
     public void checkBloodnameAdd(Person person, boolean ignoreDice) {
-        // if a non-clanner or a clanner without a phenotype is here, we can just return
+        // if a non-clan personnel or a clan personnel without a phenotype is here, we can just return
```

Comment:
"Personnel" refers to multiple people.
```suggestion
        // if person is non-clan or does not have a phenotype
```